configs/components/puppet.rb

```diff
@@ -8,6 +8,16 @@
   if platform.is_windows?
     pkg.build_requires "rubygem-win32-dir"
   end
+  # Used to compile binary translation files
+  if platform.is_osx?
+    pkg.build_requires "gettext"
+  elsif platform.is_windows?
+    pkg.build_requires "pl-gettext-#{platform.architecture}"
+  elsif platform.is_aix?
+    pkg.build_requires "http://pl-build-tools.delivery.puppetlabs.net/aix/#{platform.os_version}/ppc/pl-gettext-0.19.8-2.aix#{platform.os_version}.ppc.rpm"
```

Comment:
We also have no automated testing around pulling in actual translations at runtime yet. That would be the next big step IMO to making this safer to do, to make sure that either Puppet and friends ignore the MO files on AIX, or that they actually work, and thereby make sure they're NOT broken.